src/main/java/com/force/i18n/grammar/parser/GrammaticalLabelFileHandler.java

```diff
@@ -1205,46 +1207,38 @@ public static String convertEscapedToUnicode(String str) {
                 buf.append(c);
             } else {
                 // found an escape char '\\' at the end of the string
-                if (i >= len) {
+                if (++i >= len) {
```

Comment:
@yoikawa I noticed this changes the end of string handling, so just want to double confirm if this is correct and expected?

Reply:
yes, this is correct; this is unreported bug that `i  >= len` is never `true`